packages/upload/theme/material/vaadin-upload-styles.js

```diff
@@ -23,14 +23,7 @@ registerStyles(
       align-items: baseline;
     }
 
-    /* TODO(jouni): unsupported selector (not sure why there's #addFiles element wrapping the upload button) */
-    [part='primary-buttons'] > * {
-      display: block;
-      flex-grow: 1;
-    }
```

Comment:
Removing `flex-grow: 1` doesn't seem to affect anything, as the container uses `display: inline-flex`.